snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -148,6 +148,14 @@ class ExecutorSettings(ExecutorSettingsBase):
             "required": False,
         },
     )
+    prefix: str = field(
```

Comment:
"prefix" is a bit too generic. How about naming the purpose, e.g. "prefix_uuid"?